compose/case-document-am.yml

```diff
@@ -0,0 +1,34 @@
+---
+version: '2.4'
+
+services:
+  ccd-case-document-am-api:
+    image: "${CCD_CASE_DOCUMENT_API_USE_LOCAL-hmctspublic.azurecr.io/}ccd/case-document-am-api:${CCD_CASE_DOCUMENT_API_TAG:-latest}"
+    environment:
+      CCD_DOCUMENT_API_IDAM_KEY: "${IDAM_KEY_CASE_DOCUMENT}"
+      CASE_DOCUMENT_S2S_AUTHORISED_SERVICES: ccd_case_document_am_api,ccd_gw,xui_webapp,ccd_data,bulk_scan_processor
+      IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      IDAM_OIDC_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      S2S_URL: http://service-auth-provider-api:8080
+      IDAM_USER_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      DOCUMENT_STORE_URL: http://dm-store:8080
+      CCD_DATA_STORE_URL: http://ccd-data-store-api:4452
+      CASE_DOCUMENT_SWAGGER_AM_URL: http://ccd-case-document-am-api:4455
+      CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api:4455
+      REFORM_SERVICE_NAME: ccd-case-document-am-api
+      REFORM_TEAM: ccd
+      REFORM_ENVIRONMENT: local
+      APPINSIGHTS_INSTRUMENTATIONKEY: key
+      DOCUMENT_TTL: 3600000
+      # Uncomment this line to enable JVM debugging and uncomment the port mapping below
+      # JAVA_TOOL_OPTIONS: '-XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005'
+    ports:
+      - 4455:4455
```

Comment:
Unfortunately this port number clashes with [aca-wiremock in aac-manage-case-assignment/aca-docker](https://github.com/hmcts/aac-manage-case-assignment/blob/master/aca-docker/compose/aca.yml#L32) so it cannot be run alongside AAC/MCA images.

Reply:
I will raise a PR to modify the` aca-wiremock` port

Reply:
PR raised as https://github.com/hmcts/aac-manage-case-assignment/pull/252